api/v1beta1/network_disruption.go

```diff
@@ -327,6 +376,11 @@ func (s *NetworkDisruptionSpec) Format() string {
 	return networkDescription
 }
 
+// HasHTTPFilters return true if a custom method or path is defined, else return false
+func (s *NetworkDisruptionSpec) HasHTTPFilters() bool {
+	return s.HTTP.Method != DefaultNetworkMethodFilter || s.HTTP.Path != DefaultNetworkPathFilter
```

Comment:
```suggestion
	return s.HTTP != nil && (s.HTTP.Method != DefaultNetworkMethodFilter || s.HTTP.Path != DefaultNetworkPathFilter)
```

gotta check the HTTP pointer